### PR TITLE
[ADVAPP-1001]: Modify the iconography to be more consistent in the collapsed view of the navigation

### PR DESCRIPTION
**Pull request by kandarp-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-1001

### Technical Description

> Modify the iconography to be more consistent in the collapsed view of the navigation.

### Any deployment steps required?

> No.

### Are any Feature Flags and/or Data Migrations that can eventually be removed Added?

> No.

_______________________________________________

### CODE REVIEW

**Review by Orrison — CHANGES_REQUESTED** [review]
There are currently merge conflicts.

File: app-modules/prospect/src/Filament/Pages/RecruitmentCrmDashboard.php [hunk 1]
```diff
@@ -56,6 +56,8 @@ class RecruitmentCrmDashboard extends BaseDashboard
 
     protected static string $routePath = 'recruitment-crm-dashboard';
 
+    protected static ?string $navigationIcon = '';
```
Comment (Orrison) [inline app-modules/prospect/src/Filament/Pages/RecruitmentCrmDashboard.php:59]: Does `null` work here? Or does it have to be an empty string to not default to something?

File: app-modules/student-data-model/src/Filament/Pages/RetentionCrmDashboard.php [hunk 2]
```diff
@@ -55,6 +55,8 @@ class RetentionCrmDashboard extends BaseDashboard
 
     protected static string $routePath = 'retention-crm-dashboard';
 
+    protected static ?string $navigationIcon = '';
+
```
Comment (Orrison) [inline app-modules/student-data-model/src/Filament/Pages/RetentionCrmDashboard.php:59]: Same question as the other dashboard.

**Review by Orrison — CHANGES_REQUESTED** [review]
Some merge conflicts again.

File: app-modules/prospect/src/Filament/Pages/RecruitmentCrmDashboard.php (hunk 1 above)
Comment (kandarp-canyon) [reply app-modules/prospect/src/Filament/Pages/RecruitmentCrmDashboard.php:59]: I've tried with null as I mentioned in the call but it doesn't work, so I went with the empty string.

File: app-modules/student-data-model/src/Filament/Pages/RetentionCrmDashboard.php (hunk 2 above)
Comment (kandarp-canyon) [reply app-modules/student-data-model/src/Filament/Pages/RetentionCrmDashboard.php:59]: @Orrison null doesn't work for this.

**Review by Orrison — APPROVED** [review]

**Pull request merged**